fastchat/model/compression.py

```diff
@@ -102,26 +102,42 @@ def apply_compressed_weight(module, compressed_state_dict, target_device, prefix
 
 def load_compress_model(model_path, device, torch_dtype, use_fast, revision="main"):
     # partially load model
-    tokenizer = AutoTokenizer.from_pretrained(
-        model_path, use_fast=use_fast, revision=revision
-    )
-
+    # `use_fast=True`` is not supported for some models.
+    try:
+        tokenizer = AutoTokenizer.from_pretrained(
+        model_path, use_fast=use_fast, revision=revision,trust_remote_code=True)
+    
+    except Exception as e:
```

Comment:
Do not use too broad exception. List the specific exception.

Reply:
resolved